content/en/data_jobs/databricks.md

```diff
@@ -141,10 +143,44 @@ Optionally, you can also set other init script parameters and Datadog environmen
 
 In Datadog, view the [Data Jobs Monitoring][6] page to see a list of all your Databricks jobs.
 
-## Tag spans at runtime
+## Advanced Configuration
+
+### Tag spans at runtime
 
 {{% djm-runtime-tagging %}}
 
+### Aggregate cluster metrics from one-time job runs
+   This configuration is only applicable when you install the Datadog Agent with global init and create a new cluster for each task run.
+
+   If you are submitting Databricks Jobs via the [one-time run API endpoint][8] (common when using orchestration tools outside of Databricks such as Airflow or Azure Data Factory), you need to set the `DD_JOB_NAME` variable inside the `spark_env_vars` of every `new_cluster` to the same value as your request payload's `run_name`. This allows Data Jobs Monitoring to aggregate cluster utilization from the same job and show performance over time.
```

Comment:
Maybe start with a problem statement?
```suggestion
   If you are submitting Databricks Jobs via the [one-time run API endpoint][8] (common when using orchestration tools outside of Databricks such as Airflow or Azure Data Factory), each job run will have a unique job ID. If the job uses ephemeral clusters {{maybe there is a better way to phrase it?}} the cluster metrics will not be grouped by the job name because the job IDs are unique. You can set the `DD_JOB_NAME` variable inside the `spark_env_vars` of every `new_cluster` to the same value as your request payload's `run_name`. This allows Data Jobs Monitoring to aggregate cluster utilization from the same job and show performance over time.
```

Reply:
I think it make sense to start with the problem, update made following you recommendation.